common-arcana.lic

```diff
@@ -81,14 +81,21 @@ module DRCA
 
   def activate_barb_buff(name)
```

Comment:
It would be great to rename this to `activate_barb_buff?` now that it's a predicate. But I realize that makes it a little more complicated of a change. What are your thoughts?

Reply:
Sounds reasonable to me, and should just be a "find and replace" change. Thanks

Reply:
> should just be a "find and replace" change

Kind of. Since this PR is only for changes to the `common*` files, you'd probably want two functions (new name and old name) for a brief period of time (with one just calling the other). In the next PR they can be collapsed.

Reply:
Done, I've added new method `activate_barb_ability?` to include ? since it's a predicate now like `activate_khri?`. The previous method delegates to this new one and will be removed in a future PR.